helm/observability-bundle/values.yaml

```diff
@@ -104,7 +104,7 @@ apps:
     namespace: kube-system
     # used by renovate
     # repo: giantswarm/promtail-app
-    version: 1.4.1
+    version: 1.5.0
```

Comment:
Do you want to upgrade kube-prometheus-stack as well ?

Reply:
Can you also add the other PRs waiting here https://github.com/giantswarm/observability-bundle/pulls (prometheus-agent and grafana agent) ?

Reply:
Yeah you're right, I also added it